zeebe/protocol-impl/src/main/java/io/camunda/zeebe/protocol/impl/encoding/AuthInfo.java

```diff
@@ -103,10 +103,10 @@ public String toString() {
         + getFormat()
         + ", "
         + "authData="
-        + getAuthData()
+        + (authDataProp.isSet() ? "***" : "<unset|default>")
```

Comment:
❓ Isn;t this now doubled? Like the property is sanitized now right? So this should be handled by the property?